roles/kubernetes/control-plane/templates/kubeadm-config.v1beta2.yaml.j2

```diff
@@ -64,8 +64,8 @@ etcd:
 {% for key, value in etcd_extra_vars.items() %}
       {{ key }}: "{{ value }}"
 {% endfor %}
-{% if host_architecture != "amd64" -%}
-      etcd-unsupported-arch: {{host_architecture}}
+{% if host_architecture != "amd64" %}
+    etcd-unsupported-arch: {{host_architecture}}
```

Comment:
There is another PR that doesn't change the indentation
https://github.com/kubernetes-sigs/kubespray/pull/6919/files#diff-0be1b4e4a9316f81ccc4a70601c1f3bfbf40b85637da5507048b5f7c4fd3baf3L67-R68